File: CSharp-Part-1/3. Operators and Expressions/OperatorsAndExpressions/PointInCircle/PointInCircle.cs
// Write an expression that checks if given point (x, y) is inside a circle K({0, 0}, 2).

using System;

class PointInCircle
{
    static void Main()
    {
        double x = double.Parse(Console.ReadLine());
        double y = double.Parse(Console.ReadLine());
        bool isInside = false;
        if (Math.Pow((0 - x), 2) + Math.Pow((0 - y), 2) <=  4)
        {
            isInside = true;
        }
        Console.WriteLine(isInside);

    }
}

File: JavaScript-Fundamentals/04. Operators and Expressions/point-in-circle.js
// Write an expression that checks if given point P(x, y) is within a circle K(O, 5).

function isPointInCircle(xPoint, yPoint, xCircle, yCircle, rCircle) {
	return ((xPoint - xCircle) * (xPoint - xCircle)) + ((yPoint - yCircle) * (yPoint - yCircle)) <= rCircle * rCircle; 
}

console.log(isPointInCircle(0.9, -1.93, 0, 0, 5));